### ISSUE TITLE
Deploy Metabase frontend for BigQuery

### ISSUE THREAD

**Issue by lottspot** [cal-itp/data-infra#87]
After some discussion between @themightychris and @machow we are looking to deploy a Metabase frontend to the BigQuery database. Our plan to deploy this instance is to first deploy it to a kubernetes cluster running on GKE. The main reason we favor this approach is that a kubernetes cluster will provide a single plane of control for any future services deployed in support of the data infrastructure. E.g.:

* A separate cluster within which AirFlow's Pod Operator can run (helping avoid resource exhaustion within the composer deployment)
* Potential place for scheduled jobs to execute?
* Any other odd jobs or services which may prove helpful in the support of collection and processing

For now though, we will simply deploy the cluster with a running metabase instance which is configured to front the BigQuery database. The plan is to structure the deployment using helm charts and capture the steps through repo-embedded documentation.

Task breakdown:

- [x] Deploy GKE cluster
- [x] Document GKE cluster deployment
- [x] Deploy cluster services
- [x] Document cluster services
- [x] Deploy Metabase
- [x] Configure Metabase
- [x] Metabase backups
- [x] Document Metabase deployment

**Comment by hunterowens** [comment]
This SGTM, esp because we can also add #2 to this deployment. @lottspot can you provide a monthly estimated $ so I can plug it into our budgeting spreadsheet? 

cc'ing @thekaveman @unreasonableman for thoughts as well.

**Comment by hunterowens** [comment]
checking in on this?

**Comment by lottspot** [comment]
I've prepared a [calculator estimate](https://cloud.google.com/products/calculator#id=ba09b026-67b5-4926-969a-b63ce3cd3e16) for expected cost. Note that the main variable costs here are:

- Persistent Disk: The metabase deployment will only have need for one, but as we expand usage of the cluster, the line item in the estimate will serve as a useful guide on how persistent storage workloads will impact cost.
- Standard Storage: This line item represents the rate at which we would be charged for storage consumption in GCR. We don't actually expect to use any additional GCR space for the initial cluster + metabase deployment, but it's included as a useful future guide.
- Internet Egress: This is included as an egress estimate from GCR. Like the Storage case, we don't expect to actually incur any of these charges since most/all of the egress would be intra-regional, but just in case registry usage expands to the point where extra-regional egress comes into play, this line item also can be a base guide.

In any case, the variable costs are relatively negligible, and the bulk of the cost comes from the 3-node pool and the cluster management fee.

**Comment by lottspot** [comment]
bounced backups into a new issue (#216) so this can be closed once #141 is merged

**Comment by hunterowens** [comment]
Merged!
